CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# Version 2.2.00
+
+### New features
```

Comment:
Please make this section inline with information present in techdocs.
Copy paste if required.
https://techdocs-author.broadcom.com/content/broadcom/techdocs/us/en/ca-enterprise-software/layer7-api-management/mobile-sdk-for-ca-mobile-api-gateway/2-2/release-notes.html

Reply:
okay got it, this section is only for MAS Proximity, 
thats why its empty.
please ignore the first comment then, keep it empty